firewall/interception/windowskext/kext.go

```diff
@@ -156,9 +110,6 @@ func RecvVerdictRequest() (*VerdictRequest, error) {
 	if !ready.IsSet() {
 		return nil, ErrKextNotReady
 	}
-
-	new := &VerdictRequest{}
-
 	// wait for urgent requests to complete
```

Comment:
This was used to prioritize setting verdicts over receiving verdicts. Do we still need this? Ie. is access to the driver still locked and synchronous?